lambda/events/get/index.js

```diff
@@ -1,78 +1,78 @@
     
-    // responseBody variable to capture the response from DynamoDB for return message
-    let responseBody = '';
-    
-    // statusCode variable to capture the statusCode as result of DynamoDB action (i.e. 404, 500 )
-    let statusCode = 0;
-    
-    // parameters for DynamoDB request. 
-    // ProjectionExpression is for the request attributes of events in table 
-    const params = {
-        TableName: 'ak-api-zak-dynamodb',
-        ProjectionExpression: "event_id, user_name, activity_type, description, org_name, location_name, location_address, contact_name, contact_phone, contact_email, start_date_time, end_date_time, frequency, cost, picture_url, min_age, max_age, disability_types, event_status, approver, created_timestamp, event_link, event_name"
-    }//end of params
+   // DynamoDB.DocumentClient is a helper class for accessing DynamoDB with Javascript
+   const documentClient = new AWS.DynamoDB.DocumentClient({ region: 'us-west-2'});
+  
+   // responseBody variable to capture the response from DynamoDB for return message
+   let responseBody = '';
+  
+   // statusCode variable to capture the statusCode as result of DynamoDB action (i.e. 404, 500 )
+   let statusCode = 0;
 
-    
-    try {
-        // Creating variable {data} to store the response from DynamoDB using the params created above
-        // documentClient.scan() Returns one or more items and item attributes by accessing every item in a table.
-        const data = await documentClient.scan(params).promise();
-        
-        //setting responseBody and statusCode on successful return from DynamoDB
-        responseBody = JSON.stringify(data); 
-        statusCode = 200;
-        
-        // Log for successful DynamoDB scan
-        console.log("Successful scan. Status code: " + statusCode)
-    } catch(err) {
-        //setting responseBody and statusCode on successful return from DynamoDB
-        responseBody = 'Unable to connect to database';
-        statusCode = 500;
-        
-        // Log for error with DynamoDB scan
-        console.error("Unsuccessful connection to DynamoDB. Status code: " + statusCode);
-        console.error("Error: " + err);
-    }//end of catch
+   // Log calling API type and endpoint
+   console.log("API endpoint called: " + event.context['http-method'] + " " + event.context['resource-path']);
 
-    // Create response to return with statusCode, headers, and responseBody. 
-    const response = {
-        statusCode: statusCode,
-        headers : {
-            "Access-Control-Allow-Origin" : "*"
-        },
-        body: responseBody
-    };
-    
-    // Console log to CloudWatch for Lambda response
-    console.log("Response being returned by Lambda: ", response);
-    
-    // Console log to CloudWatch for Lambda function ending
-    console.log("ak-api-zak-lambda function ended.");
-    
-    return response;
-}//end of exports.handler
+   // Log calling API type and endpoint
```

Comment:
Technically this comment should be different than the comment on line 27. Should say something more to the effect of //Log calling IP.

Reply:
fixed